pkg/pillar/zedcloud/authen.go

```diff
@@ -178,6 +112,15 @@ func removeAndVerifyAuthContainer(ctx *ZedCloudContext, decryptCtx *cipher.Decry
 
 // VerifyAuthContainer verifies the integrity of the payload inside AuthContainer.
 func VerifyAuthContainer(ctx *ZedCloudContext, sm *zauth.AuthContainer) (types.SenderStatus, error) {
+	if ctx.serverSigningCert == nil {
+		// Load missing server cert
+		err := loadSavedServerSigningCert(ctx)
+		if err != nil {
+			ctx.log.Errorf(
+				"VerifyAuthContainer: can not load save server cert, %v\n", err)
+			return types.SenderStatusNone, err
```

Comment:
The senderStatus you return here is different than in the old code. What is the potential implact of that change? Would it make sense to preserve the old behavior?

Reply:
Forgot to address that. This changes nothing, this moves the check which always happens prior the `verifyauthcontainer` call. This was made by the following commit, which explains the purpose of the change:  https://github.com/lf-edge/eve/pull/3440/commits/589e7cd5444622bd0431e71b87c8d06191068516
```
    pillar/zedcloud: load server signing certs from verification call
    
    VerifyAuthContainer() call is senseless without loaded certs.
    Add reading of server signing certs before actual verification.
    The VerifyAuthContainer() will be called from decrypting function,
    so this change minimize code duplication.
    
```